include/psa/crypto_values.h

```diff
@@ -2034,6 +2034,8 @@
  * PSA_TLS12_ECJPAKE_TO_PMS_DATA_SIZE.
  */
 #define PSA_ALG_TLS12_ECJPAKE_TO_PMS            ((psa_algorithm_t)0x08000609)
+#define PSA_ALG_IS_TLS12_ECJPAKE_TO_PMS(alg)                               \
```

Comment:
There's no hard rule, but the idea was to have an `_IS_` macro for macros that depend on a parameter. For constants, we just use `==`.

Reply:
If we add metadata tests for `algorithm_classification` with `ALG_IS_TLS12_ECJPAKE_TO_PMS` then we need to add `PSA_ALG_IS_TLS12_ECJPAKE_TO_PMS`, as a call to it is generated by a macro - `TEST_CLASSIFICATION_MACRO`.
I'm not sure how to integrate it in a different, clean way with existing metadata tests - any suggestions, @gilles-peskine-arm?

Reply:
I don't have a strong opinion on this.
On one hand, introducing a new macro just to have some additional trivial test cases seems like an overkill, but on the other hand this way we have a standard approach for all of these algorithms. I went for the second option, since the [checklist](https://github.com/Mbed-TLS/mbedtls/blob/18e76e7eefd484c5f14d64beaeee98d70b12cee9/docs/architecture/psa-crypto-implementation-structure.md#new-functions-and-macros) suggested that `New constants must have a test case in tests/suites/test_suite_psa_crypto_metadata.data that verifies that PSA_IS_xxx macros behave properly with the new constant. New PSA_IS_xxx macros must be declared in tests/suites/test_suite_psa_crypto_metadata.function.`.
Either way is fine for me.

Reply:
The point of classification tests is to test that the `IS` macros are consistent with the value macros. Each `IS` macro should have a corresponding classification flag at the top of `test_suite_psa_crypto_metadata.function`. Each value macro should have a line in `test_suite_psa_crypto_metadata.data`. Since there is no need for `PSA_ALG_IS_TLS12_ECJPAKE_TO_PMS`, there is no need for a `ALG_IS_TLS12_ECJPAKE_TO_PMS` classification in the metadata test suite.